Update enterprise admin section (#2183)
hashicorp/terraform-website: content/enterprise/install/automated/active-active.mdx

When your organization requires increased reliability or performance from Terraform Enterprise that your current single application instance cannot provide, it is time to scale to the Active/Active architecture.

> ~> **Note:** The discontinued clustered version of Terraform Enterprise is no longer supported, and we strongly advise all customers to move to an Active/Active installation. Please contact your support representative if you need assistance.

Reviewer: ```suggestion
!> **Warning:** The discontinued clustered version of Terraform Enterprise is no longer supported, and we strongly advise all customers to move to an Active/Active installation. Please contact your support representative if you need assistance.
```

> However, the benefits that come with the Active/Active architecture need to be weighed against increasing operation complexity:

Reviewer: ```suggestion
Before scaling to Active/Active, you should weigh its benefits against increasing operational complexity. Specifically, consider the following aspects of the Active/Active architecture:
```

* Hard requirement of a completely automated installation
* Observability concerns when monitoring multiple instances
* Custom automation required to manage the lifecycle of application nodes

> * [New CLI-based commands](/enterprise/admin/infrastructure/admin) vs Replicated Admin Console for administration

Reviewer: They won't be "new" for very long! :-) 

```suggestion
* [CLI-based commands](/enterprise/admin/infrastructure/admin) for administration instead of the Replicated Admin Console
```
